sim/dex.ts

```diff
@@ -826,6 +826,22 @@ export class ModdedDex {
 				ruleTable.set(ruleSpec, '');
 				continue;
 			}
+			if (ruleSpec.split(":").length > 1) {
+				const parts = ruleSpec.split(":");
```

Comment:
```suggestion
				const [ruleName, ruleValue] = ruleSpec.split(":");
```

We generally prefer named parts. Makes it clearer what's going on.